### PR TITLE
chore(docs): add reviewer checklist PR template

### PR DESCRIPTION
**Pull request by mabdinur**
This change updates the ddtrace pull request template to include action items for reviewers. This is meant to streamline the review process and provide a set of acceptance criteria to contributors.

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

## Reviewer Checklist
- [x] PR title and description are accurate and comprehensive.The description should include motivation for the change, design decisions and concerns.
- [x] PR should be focused. No unnecessary changes were introduced in this PR.
- [x] No breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes are introduced.
- [x] Tests provided or description of manual testing performed is included in the code or PR.
- [x] Release note has been added (if necessary) and is meaningful for users.
- [x] All relevant github issues are correctly linked.
- [x] Added to the correct milestone.
- [x] Be Kind.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+
+## Reviewer Checklist
+- [ ] PR title and description are accurate and comprehensive.The description should include motivation for the change, design decisions and concerns.
+- [ ] PR should be focused. No unnecessary changes were introduced in this PR.
+- [ ] No breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes are introduced.
+- [ ] Tests provided or description of manual testing performed is included in the code or PR.
+- [ ] Release note has been added (if necessary) and is meaningful for users.
+- [ ] All relevant github issues are correctly linked.
+- [ ] Added to the correct milestone.
+- [ ] Be Kind.
```
[the remark this answers is not in the document]
Comment (mabdinur) [reply .github/PULL_REQUEST_TEMPLATE.md:15]: this is not kind

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
 <!-- Briefly describe the change and why it was required. -->
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Reviewer Checklist
+- [ ] PR title and description are accurate and comprehensive. The description should include motivation for the change, design decisions and concerns.
```
Comment (Kyle-Verhoog) [inline .github/PULL_REQUEST_TEMPLATE.md:8]:
```suggestion
- [ ] Title and description are accurate and comprehensive. The description should include motivation for the change, design decisions and concerns.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+- [ ] PR should be focused. No unnecessary changes were introduced in this PR.
```
Comment (Kyle-Verhoog) [inline .github/PULL_REQUEST_TEMPLATE.md:9]:
```suggestion
- [ ] Change should be focused. No unnecessary changes were introduced in this PR.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,7 +1,15 @@
...
+- [ ] No breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes are introduced.
```
Comment (Kyle-Verhoog) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -1,7 +1,15 @@
 <!-- Briefly describe the change and why it was required. -->
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Reviewer Checklist
```
Comment (Kyle-Verhoog) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
## Triage Reviewer Checklist
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -1,7 +1,15 @@
...
+- [ ] PR title and description are accurate and comprehensive. The description should include motivation for the change, design decisions and concerns.
+- [ ] PR should be focused. No unnecessary changes were introduced in this PR.
+- [ ] No breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes are introduced.
+- [ ] Tests provided or description of manual testing performed is included in the code or PR.
+- [ ] Release note has been added (if necessary) and is meaningful for users.
```
Comment (Kyle-Verhoog) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
- [ ] Release note has been added for fixes and features.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -1,7 +1,15 @@
...
+- [ ] All relevant GitHub issues are correctly linked.
+- [ ] Added to the correct milestone.
```
Comment (Kyle-Verhoog) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
- [ ] Add to milestone.
```

**Review by Kyle-Verhoog — DISMISSED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -1,7 +1,17 @@
 <!-- Briefly describe the change and why it was required. -->
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Triage Reviewer Checklist
+- [ ] Title and description are accurate and comprehensive. The description should include motivation for the change, design decisions and concerns.
```
Comment (majorgreys) [inline .github/PULL_REQUEST_TEMPLATE.md:8]:
```suggestion
- [ ] Title is accurate and conforms to [conventional commit](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).
- [ ] Description motivates each change.```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -1,7 +1,17 @@
 <!-- Briefly describe the change and why it was required. -->
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Triage Reviewer Checklist
```
Comment (brettlangdon) [inline .github/PULL_REQUEST_TEMPLATE.md:7]:
```suggestion
## Reviewer Checklist
```

? does it matter if it is a "Triage" or not, seems like these are just good items to check for any reviewer at all times?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 10]
```diff
@@ -1,7 +1,17 @@
...
+- [ ] Title and description are accurate and comprehensive. The description should include motivation for the change, design decisions and concerns.
+## Reviewer Checklist
+- [ ] No unnecessary changes were introduced in this PR.
+- [ ] PR cannot be broken up into smaller PRs.
+- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
+- [ ] Tests provided or description of manual testing performed is included in the code or PR.
+- [ ] Release note has been added for fixes and features.
```
Comment (brettlangdon) [inline .github/PULL_REQUEST_TEMPLATE.md:14]:
```suggestion
- [ ] Release note has been added for fixes and features, or else `changelog/no-changelog` label added.
```

?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 11]
```diff
@@ -1,7 +1,17 @@
...
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Triage Reviewer Checklist
+- [ ] Title and description are accurate and comprehensive. The description should include motivation for the change, design decisions and concerns.
+## Reviewer Checklist
```
Comment (brettlangdon) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Backports are identified and tagged with Mergifyio?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (P403n1x87) [reply .github/PULL_REQUEST_TEMPLATE.md]: An API breaking change would require a major version bump, so should this read ~ "A breaking API change is allowed only in a major version release"?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 12]
```diff
@@ -1,7 +1,17 @@
...
+- [ ] No unnecessary changes were introduced in this PR.
+- [ ] PR cannot be broken up into smaller PRs.
+- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
+- [ ] Tests provided or description of manual testing performed is included in the code or PR.
+- [ ] Release note has been added for fixes and features.
+- [ ] All relevant GitHub issues are correctly linked.
+- [ ] Add to milestone.
+- [ ] Be kind.
```
[the remark this answers is not in the document]
Comment (mabdinur) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: That checkbox is meant for Kyle

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 13]
```diff
@@ -1,7 +1,17 @@
...
+
+## Reviewer Checklist
+- [ ] Title is accurate and conforms to [conventional commit](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).
+- [ ] Description motivates each change.
+- [ ] No unnecessary changes were introduced in this PR.
+- [ ] PR cannot be broken up into smaller PRs.
+- [ ] Avoid breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
+- [ ] Tests provided or description of manual testing performed is included in the code or PR.
+- [ ] Release note has been added for fixes and features, or else `changelog/no-changelog` label added.
+- [ ] All relevant GitHub issues are correctly linked.
```
Comment (mabdinur) [inline .github/PULL_REQUEST_TEMPLATE.md:15]:
```suggestion
- [ ] All relevant GitHub issues are correctly linked.
- [ ] Backports are identified and tagged with Mergifyio
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 14]
```diff
@@ -1,7 +1,17 @@
...
+- [ ] Add to milestone.
+- [ ] Be kind Kyle.
```
Comment (mabdinur) [inline .github/PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 15]
```diff
@@ -1,7 +1,17 @@
 <!-- Briefly describe the change and why it was required. -->
 
 ## Checklist
-- [ ] Added to the correct milestone.
-- [ ] Tests provided or description of manual testing performed is included in the code or PR.
 - [ ] Library documentation is updated.
 - [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
+
+## Reviewer Checklist
+- [ ] Title is accurate and conforms to [conventional commit](https://github.com/conventional-changelog/commitlint/tree/master/%40commitlint/config-conventional).
```
Comment (majorgreys) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: The reviewer should not have to worry about what has been automated.

```suggestion
- [ ] Title is accurate.
```

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**